### PR TITLE
Problem: Selector allowed choosing a CRN with no confidential support

### PR DESCRIPTION
**Pull request by olethanh**

### CODE REVIEW

**Review by Psycojoker — APPROVED** [review]

File: src/aleph_client/commands/instance/__init__.py [hunk 1]
```diff
@@ -235,13 +235,22 @@ def validate_ssh_pubkey_file(file: Union[str, Path]) -> Path:
...
                 # User has ctrl-c
                 return
+
+            if confidential and crn.confidential_computing is False:
+                print("Choosen CRN do not support confidential computing. Please select another one.")
+                crn = None
+                continue
+            if hypervisor == HypervisorType.qemu and crn.qemu_support is False:
+                print("Choosen CRN do not support QEMU hypervisor. Please select another one.")
+                crn = None
```
Comment (philogicae) [inline src/aleph_client/commands/instance/__init__.py:245]: Missing "continue" after this line?

File: src/aleph_client/commands/instance/__init__.py [hunk 2]
```diff
@@ -235,13 +235,22 @@ def validate_ssh_pubkey_file(file: Union[str, Path]) -> Path:
             if not crn:
                 # User has ctrl-c
                 return
+
+            if confidential and crn.confidential_computing is False:
+                print("Choosen CRN do not support confidential computing. Please select another one.")
```
Comment (hoh) [inline src/aleph_client/commands/instance/__init__.py:240]:
```suggestion
                print("Selected CRN do not support confidential computing. Please select another one.")
```

**Comment by hoh** [comment]
This branch still displays nodes with no confidential support in the selector. Is that the same issue ?

**Comment by philogicae** [comment]
I close this PR, I retrieved the changes in that branch https://github.com/aleph-im/aleph-client/pull/253

**Pull request closed without merging**